transit/models/item.py

```diff
@@ -40,11 +40,11 @@ class Meta:
 
 
 class ItemDetails(BaseModel):
-    batch_number = models.CharField(db_column='BatchNumber', max_length=255, null=False)
+    batch_number = models.CharField(db_column='BatchNumber', max_length=255, null=False, blank=False)
 
-    expiry_date = models.DateTimeField(db_column='ExpiryDate', null=True)
-    manufacturing_date = models.DateTimeField(db_column='ManufacturingDate', null=True)
-    received_date = models.DateTimeField(db_column='ReceivedDate', null=True)
+    expiry_date = models.DateTimeField(db_column='ExpiryDate', null=True, blank=True)
```

Comment:
Note: We should later on change dates and decimal fields to null. For now I'll merge this to be sure that frontend will work properly.

Reply:
yes. We need to change as we discussed.